tools/annotation_tools/turk_with_s3/run_tool_1D.py

```diff
@@ -68,7 +107,7 @@
 # Postprocess
 print("*"*50)
 print("*** Postprocessing results ***")
-rc = subprocess.call(["python3 parse_tool_D_outputs.py"], shell=True)
+rc = subprocess.call([f"python3 parse_tool_D_outputs.py --folder_name {default_write_dir}/D/"], shell=True)
 if rc != 0:
     print("Error collating answers. Exiting.")
     sys.exit()
```

Comment:
Thanks for cleaning these up, we could even rewrite the subprocess calls to files like: `parse_tool_D_outputs.py`

This version definitely needed rewriting. The work I had done here was to just make sure the pre existing version could work with Turk.

Reply:
created an issue for that #804. We can tackle it when have free cycle.